<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_customer</class>
 <widget class="QDialog" name="search_customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 48);
</string>
  </property>
  <widget class="QGroupBox" name="INFO_OUT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>511</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);</string>
   </property>
   <property name="title">
    <string>Інформація</string>
   </property>
   <widget class="QSplitter" name="Second_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="second_name_L">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>187</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>313</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Прізвище:</string>
     </property>
    </widget>
    <widget class="QLabel" name="second_name_out">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="Date_of_birth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="data_L">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>187</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>313</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Дата народження:</string>
     </property>
    </widget>
    <widget class="QLabel" name="data_out">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="NIk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="nik_L">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>187</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>313</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Нікнейм:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nik_out">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="EMAIL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="email_L">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>187</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>313</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Електронна пошта:</string>
     </property>
    </widget>
    <widget class="QLabel" name="email_out">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="First_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="name_L">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>187</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>313</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Ім'я:</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_out">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="ID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="ID_L">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>187</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>313</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ID_out">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>698</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Пошук Користувача</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_B">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="INFO_IN">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>221</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 90);</string>
   </property>
   <property name="title">
    <string>Введіть дані:</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="name_in_L">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Ім'я:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_in">
     <property name="maximumSize">
      <size>
       <width>117</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);


</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="second_name_in_L">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>Прізвище:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="second_name_in">
     <property name="maximumSize">
      <size>
       <width>117</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);


</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="ID_IN_L">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_IN">
     <property name="maximumSize">
      <size>
       <width>117</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color:rgb(0, 0, 140);


</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="search_B">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);


</string>
   </property>
   <property name="text">
    <string>Пошук</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear_B">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgb(0, 0, 140);


</string>
   </property>
   <property name="text">
    <string>Очистити</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>326</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="stat_L">
    <property name="maximumSize">
     <size>
      <width>84</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
    </property>
    <property name="text">
     <string> Статус: </string>
    </property>
   </widget>
   <widget class="QLabel" name="stat_info">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>238</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(0, 0, 140);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
